events/static/events/create_event.css

```diff
@@ -0,0 +1,245 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&display=swap");
+
+* {
+  transition: 0.1s ease;
+}
+
+body {
+  background: url("https://images.unsplash.com/photo-1480497490787-505ec076689f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1738&q=80") no-repeat;
+  background-size: cover;
```

Comment:
I have seen that the background does not cover the whole screen in some cases.
Please change the background-size to auto instead of cover.

Reply:
background covers the whole screen now